recipes/WSJ0Mix/meta/wham_room.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
```

Comment:
Please add a general docstring + a function specific one. If this is taken from somewhere else, it should be mentioned a credits should be given.